libs/components/src/lib/data-grid/data-grid-cell.scss

```
@@ -31,11 +33,15 @@
 	display: flex;
 	box-sizing: border-box;
 	align-items: center;
-	padding: 8px 12px;
+	padding: 14px 12px;
```

```suggestion
	padding: 12px;
```
14px is not on the 8px scale...

---

The value is taken from the current design in Figma. I've asked Ayala to comment on it.
However, I don't think it really matters, as changing the margin would be a breaking change and is unrelated to this ticket.

---

I don't think it will be a breaking change as cell has min-height of 48px, so it can be either way.
And only if its bigger then 48px then the padding make a difference.

---

Yes, but I think the padding should always be the same

---

@AyalaBu ?

---

I think that top margin and bottom margin should stay as they are right now,
Even the text has multiple lines.
If the text "jumps" while filling the table is a very unpleasant experience
And also it there are a few cells one near another with different amount of content and different top margin is very unpleasant to the eye.